pkg/backup/backup.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 the Velero contributors.
+Copyright 2021 the Velero contributors.
```

Comment:
We use `Copyright the Velero contributors.` now.

Reply:
The [contributing guide code standards](https://velero.io/docs/v1.6/code-standards/#copyright-header) state a year is required. If this should be changed, perhaps a separate PR should update the code standard and update the header across the project.

Reply:
Thank you for the information, yes we should update the doc.

Reply:
#3830